babel/src/board.js

```diff
@@ -4,7 +4,7 @@ import * as orbs from './orbs';
 import * as triples from './triples';
 
 export class Board {
-    constructor (width = 8, height = 8, types = _.range(7)) {
+    constructor (width = 8, height = 8, types = _.range(7), atticOrbs = null) {
```

Comment:
`null` is special. I'd recommend using it only to really, really mean "something that is, but is explicitly _nothing_".

`undefined` is a more appropriate term here, as it's the value used to represent an unused default value. But in your case, you already have a default value defined below. You should use it.

```js
let attic = _.cloneDeep(new Board(this.width, this.height, this.types, true).orbs);
export class Board {
    constructor (width = 8, height = 8, types = _.range(7), atticOrbs = attic) {
        //...
    }
}
```